### PR TITLE
Spelling and formatting, minor changes to wording

### PR DESCRIPTION
**Pull request by meyer-chris**
Spelling and formatting, minor changes to wording

### CODE REVIEW

**Comment by henkbirkholz** [comment]
A few questions:
* why the line breaks in the header?
* why adding some brackets in some places (e.g., Hannes' email address but not Mike's)
* what's the intended line break pattern in the kramdown part? (the intend could  be per paragraph, but then there would be quite a few exception to that pattern)

Personally, I'd prefer an editorial datatracker submission that would transition to a "per sentence" line break pattern & no only necessary line breaks in the header.

**Comment by meyer-chris** [comment]
Fair points, Henk. And I must confess, For the brackets, I have no clue :-)
- Brackets seem to have been added by the auto-format function of my editor. 
- Line breaks sentences was something I removed to make Markdown exports more targeted if required, but nothing of major importance. If we want to do it differently fine as well, then we need to limit line lengths to 72 characters if I'm not mistaken.

Formatting is less important to me, the spelling fixes are something that we should merge, I reckon.

**Comment by ounsworth** [comment]
Note-to-self: I'll need to pull this locally and go through the diffs one-by-one.

File: draft-ietf-lamps-csr-attestation.md [hunk 1]
```diff
@@ -55,93 +66,50 @@ informative:
...
+A client requesting a certificate from a Certification Authority (CA) may wish to offer believable claims about the protections afforded to the corresponding private key, such as whether the private key resides on a hardware security model or trusted platform module, and the protection capabilities provided by the hardware module. Including this evidence along with the certificate request can help to improve the assessment of the security posture for the private key, and suitability of the submitted key to the requested certificate profile. These evidence claims can include information about the hardware component's manufacturer, the version of installed or running firmware, the version of software installed or running in layers above the firmware, or the presence of hardware components providing specific protection capabilities or shielded locations (e.g., to protect keys).
```
Comment (jethrogb) [inline draft-ietf-lamps-csr-attestation.md:79]:
```suggestion
A client requesting a certificate from a Certification Authority (CA) may wish to offer believable claims about the protections afforded to the corresponding private key, such as whether the private key resides on a hardware security module or trusted platform module, and the protection capabilities provided by the hardware module. Including this evidence along with the certificate request can help to improve the assessment of the security posture for the private key, and suitability of the submitted key to the requested certificate profile. These evidence claims can include information about the hardware component's manufacturer, the version of installed or running firmware, the version of software installed or running in layers above the firmware, or the presence of hardware components providing specific protection capabilities or shielded locations (e.g., to protect keys).
```

**Comment by ounsworth** [comment]
@meyer-chris This PR is 98% whitespace changes. I think I have extracted the meaningful content changes and submitted that as #43 .

If you're ok with it, I'd like to merge #43 and close this PR.

**Comment by meyer-chris** [comment]
Hey Mike, absolutely, feel free to go ahead! Fine by me... As mentioned, just a few typos

**Pull request closed without merging**